docs/data-sources/secmaster_workspaces.md

```diff
@@ -77,3 +77,19 @@ The `workspaces` block supports:
 * `created_at` - The creation time.
 
 * `updated_at` - The update time.
+
+* `version` - The version of the data source of an alert.
+
+* `domain_id` - The ID of the account (domain_id) to whom the data is delivered and hosted.
+
+* `project_id` - The ID of project where the account to whom the data is delivered and hosted belongs to.
+
+* `region_id` - The ID of the region where the account to whom the data is delivered and hosted belongs to.
+
+* `workspace_id` - The ID of the current workspace.
+
+* `arrive_time` - The data receiving time.
+
+* `count` - The times of the alert occurrences.
+
+* `data_class_id` - The data class ID.
```

Comment:
why add these attributes, they are not added in the data source

Reply:
Submitted by mistake